### PR TITLE
Add revenue tracking instructions for Events API

### PR DESCRIPTION
**Pull request by vinibrsl**

### CODE REVIEW

File: docs/events-api.md [hunk 1]
```diff
@@ -124,6 +124,17 @@ The value corresponding to the `props` key in the request body is expected to be
 ```
 <hr / >
 
+**revenue** <Optional />
+
+Revenue data for this event. This can be attached to goals and custom events to track revenue attribution. To learn more about revenue tracking and how you can set up your dashboard before sending revenue data, please read [this document](ecommerce-revenue-tracking.md).
+
+The value corresponding to the `revenue` key in the request body is expected to be a JSON object with `currency` and `amount`. `currency` is a ISO 4217 string representing the currency code, e.g. `"USD"` or `"EUR"`. `amount` can be either a float or a string, e.g. `1322.22` or `"1322.22"`. For example, to track a purchase of US$ 1.322,22, your request body might look like this:
```
Comment (ukutaht) [inline docs/events-api.md:131]: What happens if currency or amount is invalid? Event rejected, plus bad response code? Silently ignored?

Comment (vinibrsl) [reply docs/events-api.md]: Added a note in f36ce30

Comment (vinibrsl) [reply docs/events-api.md]: Added a note in 0078392 ;)

**Comment by ukutaht** [comment]
Thanks @vinibrsl ❤️

**Pull request merged**